app/common/auth/__init__.py

```diff
@@ -104,34 +104,37 @@ def sso_sign_in() -> ResponseReturnValue:
 @auto_commit_after_request
 def sso_get_token() -> ResponseReturnValue:
```

Comment:
Nice refactor of this endpoint - commenting and separating out the main branches makes it feel easier to reason about, ty 🙇